lib/fluent/plugin/out_kafka_buffered.rb

```diff
@@ -339,7 +340,8 @@ def write(chunk)
         end
         log.trace { "message will send to #{topic} with partition_key: #{partition_key}, partition: #{partition}, message_key: #{message_key} and value: #{record_buf}." }
         messages += 1
-        producer.produce_for_buffered(record_buf, topic: topic, key: message_key, partition_key: partition_key, partition: partition)
+        producer.produce_for_buffered(record_buf, topic: topic, key: message_key, partition_key: partition_key, partition: partition
```

Comment:
it needs `,` after `partition: partition`

Reply:
good catch!